models/store/src/store/store-draft/presets/sample-data-goodstore/the-good-store.ts

```diff
@@ -0,0 +1,15 @@
+import { LocalizedString } from '@commercetools-test-data/commons';
+import type { TStoreDraftBuilder } from '../../../types';
+import * as StoreDraft from '../../index';
+
+const the_good_store = (): TStoreDraftBuilder =>
+  StoreDraft.presets
```

Comment:
The original data includes `countries` and `languages`, do we want to include those here?

Reply:
Since we have only one store, and it supports all the languages and all the countries, is it necessary?

Reply:
The API assumes all, unless stated otherwise